aspnet/fundamentals/hosting.rst

```diff
@@ -0,0 +1,297 @@
+:version: 1.0.0
+
+Hosting
+=======
+
+By `Steve Smith`_
+
+To run an ASP.NET Core app, you need to configure and launch a host using `WebHostBuilder <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNetCore/Hosting/WebHostBuilder/index.html?highlight=webhostbuilder>`_.
+
+.. contents:: Sections:
+  :local:
+  :depth: 1
+
+What is a Host?
+---------------
+
+ASP.NET Core apps require a *host* in which to execute. A host must implement the :dn:iface:`~Microsoft.AspNetCore.Hosting.IWebHost` interface, which exposes collections of features and services, and a ``Start`` method. The host is typically created using an instance of a :dn:class:`~Microsoft.AspNetCore.Hosting.WebHostBuilder`, which builds and returns a  :dn:class:`~Microsoft.AspNetCore.Hosting.Internal.WebHost` instance. The ``WebHost`` has a `private Server property <https://github.com/aspnet/Hosting/blob/1.0.0/src/Microsoft.AspNetCore.Hosting/Internal/WebHost.cs#L40>`_ that specifies the server that will handle requests. Learn more about :doc:`servers <servers>`.
+
+What is the difference between a host and a server?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The host is responsible for ensuring the application's services and the server are available and properly configured. You can think of the host as being a wrapper around the server. The host is configured to use a particular server; the server is unaware of its host.
```

Comment:
The Host is responsible for application startup and lifetime management, where the server is responsible for accepting HTTP requests.